dap_prinz_green_jobs/tests/test_skill_measures.py

```diff
@@ -0,0 +1,86 @@
+import pytest
```

Comment:
all tests passed with `9 passed, 45 warnings` 🤔 mostly `DeprecationWarning` warnings across numpy and scipy.

Reply:
I'm on 15 warnings. It's not clear to me what's causing them apart from package versions?